### PR TITLE
SampleGen: Add an await pattern for LROs in Node.js

### LINKED ISSUE #2579: SampleGen: Node.js blockers
### 🚀 Node.js Blocking Issues

 - [x] Missing `require` import
 - [x] Missing service client instantiation
 - [x] Sample is currently generated commented out
 - [ ] `require(`yargs`)` needs to be `const argv = require(`yargs`)`
 - [ ] String interpolation needs to use `${var}`
 - [ ] LRO needs to work

Example of Node.js output below:

_(I removed the wrapping comment block for proper syntax highlighting)_

```js
//// [ This is an auto-generated sample file produced by the gapic-generator. Sample name: "AnalyzeSyntaxRequestLanguageSyntaxText" ]
//// STUB standalone sample "AnalyzeSyntaxRequestLanguageSyntaxText" /////

// FIXME: Insert here set-up comments that we never want to display in cloudsite. These are seen by users perusing the samples directly in the repository.

// [START language_syntax_text]

// FIXME: Insert here boilerplate code not directly related to the method call itself.

//      calling form: "Request"
//        region tag: "language_syntax_text"
//         className: "AnalyzeSyntaxRequestLanguageSyntaxText"
//          valueSet: "language_syntax_text" ("")
//       description: "Analyze syntax of text"
//        [document.type=PLAIN_TEXT, document.content="This is a short sentence."]
//      apiMethod "analyzeSyntax" of type "OptionalArrayMethod"

// [START language_syntax_text_core]

// FIXME: Insert here code to prepare the request fields, make the call, process the response.

// /*
function sampleAnalyzeSyntax(textContent) {
  // const textContent = 'This is a short sentence.';
  const type = 'PLAIN_TEXT';
  const document = {
    type: type,
    content: textContent,
  };
  client.analyzeSyntax({document: document})
    .then(responses => {
      const response = responses[0];
      const tokens = response.tokens;
      for (const token of tokens) {
        console.log(`Part of speech: {$token.partOfSpeech.tag}`);
        console.log(`Text: {$token.text.content}`);
      }
    })
    .catch(err => {
      console.error(err);
    });
}

require(`yargs`)
  .default('text_content', 'This is a short sentence.')
  .argv;

sampleAnalyzeSyntax(argv.text_content);
// */
// [END language_syntax_text_core]

// FIXME: Insert here clean-up code.
// tslint:disable-next-line:no-any
// [END language_syntax_text]
```

### PR DESCRIPTION
**Pull request by hzyi-google**
Fix #2579

### CODE REVIEW

**Comment by hzyi-google** [comment]
cc/ @beccasaurus

File: src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline [hunk 1]
```diff
@@ -481,6 +481,77 @@ function sampleGetBigBook(shelf, bigBookName) {
...
+
+// [END hopper_core]
+// [END hopper]
+// tslint:disable-next-line:no-any
+
+const argv = require(`yargs`)
+  .default('shelf', 'Novel')
+  .argv;
+
+sampleGetBigBook(argv.shelf);
```
Comment (alexander-fenster) [inline src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline]: can we add `.catch(console.error)` to highlight that this is an async function?

Comment (hzyi-google) [reply src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline]: Sure!

**Comment by hzyi-google** [comment]
Oh, forgot to mention, the templates are derived from [here](https://github.com/googleapis/nodejs-speech/blob/7d16184c2692ed806eaba6deaf1ba7a08cf3faa5/samples/recognize.js#L210-L215), I _think_ this is the pattern @beccasaurus wants, but she needs to confirm.

File: src/main/resources/com/google/api/codegen/nodejs/standalone_sample.snip [hunk 2]
```diff
@@ -238,6 +245,18 @@
     });
 @end
 
+@private methodCallSampleCodeLongRunningPromiseAwait(apiMethod, sample)
+  {@initCode(apiMethod, sample.sampleInitCode)}
+
+  // Create a job that you can wait for now, or get its result later
```
Comment (vchudnov-g) [inline src/main/resources/com/google/api/codegen/nodejs/standalone_sample.snip]: "Create a job whose results you can either wait for now, or get later"

Comment (vchudnov-g) [reply src/main/resources/com/google/api/codegen/nodejs/standalone_sample.snip]: BTW, I assume initial capitalization and no final period is the comment style we're going for?

Comment (hzyi-google) [reply src/main/resources/com/google/api/codegen/nodejs/standalone_sample.snip]: idk, I literally copied the comments over from the example Rebecca gave me :/

Comment (vchudnov-g) [reply src/main/resources/com/google/api/codegen/nodejs/standalone_sample.snip]: I would still tweak the wording as per above. The original was not quite structurally clear English....

Comment (hzyi-google) [reply src/main/resources/com/google/api/codegen/nodejs/standalone_sample.snip]: Oh yeah for sure.

**Comment by hzyi-google** [comment]
PTAL @beccasaurus @alexander-fenster

**Pull request merged**